zuul.d/ansible-network-jobs.yaml

```diff
@@ -0,0 +1,10 @@
+---
+
+- job:
+    name: network-ee-integration-tests-ansible-network
+    parent: ansible-network-ios-appliance
```

Comment:
This isn't correct, if we parent to ios, then all testing will use this image.  Do you plan to run tests against eos for example?

Reply:
> This isn't correct, if we parent to ios, then all testing will use this image. Do you plan to run tests against eos for example?

yes we might want to add tests for multiple platforms in future

Reply:
The way this is done now, will not work as you expect. I would like you to review and even try to land https://github.com/ansible-collections/cisco.ios/pull/397

This will give you a better understanding of how the new network-ee jobs work.

Basically, this job would just run cisco.ios tests, not the ansible.network collection.  For that to happen, we need to setup collection_name / collection_namespace in the inventory file

Reply:
@pabelanger so are you suggesting having a specific job like `ansible-test-network-integration-ios-python36`
because for now, we are testing for ios.